content/docs/instrumenting/clientlibs.md

```diff
@@ -21,6 +21,7 @@ HTTP endpoint on your application’s instance:
 Unofficial third-party client libraries:
 
 * [Bash](https://github.com/aecolley/client_bash)
+* [Clojure](https://github.com/soundcloud/prometheus-clj) (wraps the Java client library)
```

Comment:
@brian-brazil Feel free to review. I put it into the unofficial section because I wanted to spare us the work. We were pretty relaxed with including links as long as we marked them as unofficial, i.e. not necessarily following approved best practices etc.